cli/src/main/java/org/projectnessie/tools/catalog/migration/cli/BaseRegisterCommand.java

```diff
@@ -0,0 +1,359 @@
+          operated(),
+          result.registeredTableIdentifiers().size(),
+          sourceCatalogOptions.type.name(),
+          targetCatalogOptions.type.name());
+    }
+    if (!result.failedToRegisterTableIdentifiers().isEmpty()) {
+      consoleLog.error(
+          "Failed to {} {} tables from {} catalog to {} catalog. "
+              + "Please check the `catalog_migration.log` file for the failure reason. "
+              + "Failed identifiers are written into `{}`. "
+              + "Retry with that file using `--identifiers-from-file` option "
+              + "if the failure is because of network/connection timeouts.",
+          operate(),
+          result.failedToRegisterTableIdentifiers().size(),
+          sourceCatalogOptions.type.name(),
+          targetCatalogOptions.type.name(),
+          FAILED_IDENTIFIERS_FILE);
+    }
+    if (!result.failedToDeleteTableIdentifiers().isEmpty()) {
+      consoleLog.error(
+          "Failed to delete {} tables from {} catalog. "
+              + "Please check the `catalog_migration.log` file for the failure reason. "
+              + "{}Failed to delete identifiers are written into `{}`.",
+          result.failedToDeleteTableIdentifiers().size(),
+          sourceCatalogOptions.type.name(),
+          System.lineSeparator(),
+          FAILED_TO_DELETE_AT_SOURCE_FILE);
+    }
+  }
+
+  private void printDetails(CatalogMigrationResult result) {
+    consoleLog.info("Details: ");
+    if (!result.registeredTableIdentifiers().isEmpty()) {
+      consoleLog.info(
+          "Successfully {} these tables:{}{}",
+          operated(),
+          System.lineSeparator(),
+          result.registeredTableIdentifiers());
+    }
+
+    if (!result.failedToRegisterTableIdentifiers().isEmpty()) {
+      consoleLog.error(
+          "Failed to {} these tables:{}{}",
+          operate(),
+          System.lineSeparator(),
+          result.failedToRegisterTableIdentifiers());
+    }
+
+    if (!result.failedToDeleteTableIdentifiers().isEmpty()) {
+      consoleLog.error(
+          "Failed to delete these tables from source catalog:{}{}",
+          System.lineSeparator(),
+          result.failedToDeleteTableIdentifiers());
+    }
+  }
+
+  private void printDryRunResult(Set<TableIdentifier> result) {
+    consoleLog.info("Summary: ");
+    consoleLog.info(
+        "Identified {} tables for {} by dry-run. These identifiers are also written into {}. "
+            + "This file can be used with `--identifiers-from-file` option for an actual run.",
+        result.size(),
+        operation(),
+        DRY_RUN_FILE);
+    consoleLog.info(
+        "Details: {}Identified these tables for {} by dry-run:{}{}",
+        System.lineSeparator(),
+        operation(),
+        System.lineSeparator(),
+        result);
+  }
+
+  private static void writeToFile(Path filePath, Collection<TableIdentifier> identifiers) {
+    if (identifiers.isEmpty()) {
```

Comment:
When we discussed this tool in detail, we concluded that the purpose of the "failed" and "successful" table-identifier list is to allow retries with the failed tables. 

This condition however likely makes that scenario impossible and leads to ambiguous results - the same table identifier appearing in the files for successful and failed migrations.

Reply:
True. 
Removed now.